### DISCUSSION TITLE
GitHub Contributions not showing after account merge

### DISCUSSION THREAD

**Discussion by bryanwhl** [community/community#32856] [Other Feature Feedback, Questions, & Ideas]
Hey everyone, not sure how to resolve this, but I spent a few hours trying to get my contribution to show but it just doesn't. I've went through the checklist and ensured the conditions are all met in this "Why are my contributions not showing" [article](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile), and I've also followed the steps in this "Merging multiple accounts" [article](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts), but yet my contributions for the past 9 months on a github organization is still not shown on my profile.

For context, I've been contributing to a few repositories on a GitHub organization on another GitHub account for the past 9 months, and I had just merged that other github account into this one by deleting it and adding the email associated to this account. I've also been invited to that organization on this account, and I've done a git push successfully here. However, after more than 10 hours of waiting, the GitHub contribution to that organization is still not showing, despite the commits' author being linked to my current account already (i.e. it's already been recognized that the past 9 months worth of commit is tied to this account, and yet the contribution is not updated).

Can someone tell me how to go about getting the contributions to show?

**Comment by RedoxOwn** [comment]
The fastest and safest way should be to Contact GitHub.
They should be able to fix your issue for you.

**Comment by SharfShiplo** [comment]
Your local Git commit email isn't connected to your account
Commits must be made with an email address that is connected to your account on GitHub.com, or the GitHub-provided noreply email address provided to you in your email settings, in order to appear on your contributions graph.

**Comment by jfullstackdev** [comment]
is the email from the other the real email ? if that is the `noreply`, then that might be a problem, you can still wait for 24 hours too, let's see

Staff members here might see this post but you can also try, after 24 hours, contacting the Support, although there is still no assurance for that

https://support.github.com/contact/account

**Reply by bryanwhl** [reply]
It's the other real email, not the noreply email, which is why I thought it would work for sure.

**Reply by RaulGB88** [reply]
I have the same problem.
I delete the old account, and put this email in my personal account.
Now I am in the organization but I cant' see my old contributions.
This is a problem in Github.
